safe-global/safe-wallet-web
# Feat(Multichain): Show warning on dashboard when multichain setup is inconsistent [SW-96]

## What it solves

Resolves: https://www.notion.so/safe-global/Multichain-Safes-V1-view-owners-2b50fadd1f6c4abe9ad836461353b98d

## How this PR fixes it
- Checks if there are other safes in the group that differ in their owner and threshold setup from the currently opened safe.
- If there are deviations, show a warning message on the dashboard. The message contains the names of the chains where the setup is different from the current safe.

## How to test it
- Create a multichain safe
- For some of the chains in the group, change the owners or threshold.
- When on the dashboard of any of the safes in the group, the message should be shown, specifying all deviating chains.

## Screenshots
![image](https://github.com/user-attachments/assets/eb66981f-7170-4678-9911-2e6e131d3c91)


## Checklist
